src/QueryType/Select/Query/FilterQuery.php

```diff
@@ -136,6 +136,58 @@ public function setTags(array $tags): self
         return $this;
     }
 
+    /**
+     * Cache the filter query or not.
+     *
+     * @param bool $cache
+     *
+     * @return self Provides fluent interface
+     */
+    public function setCache(bool $cache): self
+    {
+        $this->getLocalParameters()->setCache($cache);
+
+        return $this;
+    }
+
+    /**
+     * Get the information if the filter query should be cached or not.
+     *
+     * @return bool
+     */
+    public function getCache(): bool
+    {
+        $cache = $this->getLocalParameters()->getCache();
+        // The default is to cache the filter Query.
+        return ([] === $cache || 'true' === key($cache)) ? true : false;
```

Comment:
will `'true' === key($cache)` ever be true? I thought `->getValues(...)` will return the keys only?

https://github.com/solariumphp/solarium/blob/6a7aad5227e1462cd220e0922f155898e75ca73c/src/Core/Query/LocalParameters/LocalParameter.php#L112

Reply:
See https://github.com/solariumphp/solarium/pull/746/files#diff-3c3f87c60f9b027de6678593757504a6R170